core/db/default/spree/stores.rb

```diff
@@ -1,10 +1,12 @@
 # Possibly already created by a migration.
-unless Spree::Store.where(code: 'spree').exists?
+unless Spree::Store.default.present?
   Spree::Store.new do |s|
-    s.code              = 'spree'
     s.name              = 'Spree Demo Site'
-    s.url               = Rails.application.routes.default_url_options[:host] || 'example.com'
+    s.code              = 'spree'
+    s.url               = Rails.application.routes.default_url_options[:host] || 'demo.spreecommerce.org'
     s.mail_from_address = 'spree@example.com'
     s.default_currency  = 'USD'
+    s.seo_title         = 'Spree Commerce Demo Shop'
+    s.meta_description  = 'Spree Commerce is an open source Ecommerce framework decision makers want, developers enjoy.'
```

Comment:
Maybe we could explicitly set `default` value to true?

Reply:
@piotrleniec-spark we don't need to - https://github.com/spree/spree/blob/84e6551a395c7c26e1ea3dae1a349521608890bf/core/app/models/spree/store.rb#L11